WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt

```diff
@@ -211,6 +226,7 @@ class MySiteFragment : Fragment(),
         super.onCreate(savedInstanceState)
         setHasOptionsMenu(true)
         (requireActivity().application as WordPress).component().inject(this)
+        uiScope = CoroutineScope(uiDispatcher + job)
```

Comment:
I used this a bit hacky solution here, since MySiteFragment is going away soon so I didn't want to spent more time on it. Wdyt, is it ok?

Reply:
I'm fine with this hack. Let's keep it as you said MySiteFragment is going away soon.